docs/install/index.md

```diff
@@ -10,251 +10,22 @@ html_meta:
 
```

Comment:
Change the `keywords` above to align with the removal of the content that it references.

Reply:
@stevepiercy Keywords updated